CHANGELOG.md

```diff
@@ -1777,7 +1777,7 @@ For instance, you can collect feedback, whenever convenient as shown in this exa
 
 ```js
 const eventId = Sentry.captureMessage('User Feedback');
-const user = Sentry.getCurrentHub().getScope().getUser();
+const user = Sentry.getCurrentScope().getUser();
```

Comment:
I wouldn't change this here because 7.47.0 didn't have this API yet.

Reply:
ah, search and replace error, oops! 👍